### DISCUSSION TITLE
Yüzücüler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#246] [Zeka Sorusu]
Ali, Mete, Ozan ve Bora dört yüzücüdür. Her biri ayrı stilde (serbest, sırtüstü , kurbağalama, kelebek) yüzmektedir. Onları izleyen biri ¸su ifadeleri kullanmaktadır:
• Bora kurbağalama yüzmez.
• Ozan serbest yüzmez.
• Bora serbest yüzer.
• Mete kurbağalama yüzer.
• Ali sırtüstü yüzmez.
• Ozan kurbağalama yüzmez.
Bu ifadelerden sadece ve sadece , tam olarak, kesinlikle, ve yalnızca ( 🙂 ) biri doğru olduğuna göre kim nasıl yüzer söyleyin bakalım.

**Comment by Adaisland** [comment]
Bu yönergelerde eğerki iki tane farklı fiile( yüzmez,yüzer) sahip kişi seçersek ozan iki dalda yüzer olarak sonuç alıyoruz ve Merti seçersekte bora ile çakışıyor ve aliyi seçersekte gine ozan iki dalda yüzer çıkıyor o yüzden ozanın yönergelerinden"kurbağalama yüzmezi seçiyoruz ki boranın yanlış ifadesi olan kurbağalama yüzmez le çakışmasın

Ozan:Serbest
Bora:Kurbağalama
Mert:kelebek
Ali:Sırtüstü

**Comment by akgembi12** [comment]
İlk yönerge ve son yönergeden başka bir seçeneği secersek otomatik olarak onlar cakisacak. Bu sayede ihtimali ikiye indirdik. 2. Ve sonuncu dışında bir seçeneği secersek Dr yine aynı şekilde birbiriyle çakışacaklar. İki 2 çakışma senaryosunun da gerçekleşmemesi için sonuncu seçeneği seçiyoruz
Sonra da son seçeneğe göre bir tablo oluşturup bütün yüzücülerin stillerini şöyle buluyoruz:
Ali sırtüstü
mete kelebek
Ozan serbest
Bora kurbağa
